packages/wallet-sdk/src/mocks/relay.ts

```diff
@@ -143,10 +142,6 @@ export class MockRelayClass extends WalletSDKRelayAbstract {
     return;
   }
 
-  connectAndSignIn(): CancelablePromise<ConnectAndSignInResponse> {
```

Comment:
remove `connectAndSignIn` from abstract class.
`connectAndSignIn` will be defined only on MobileRelay for now.